egs/wsj/asr1/conf/tuning/decode_pytorch_transformer_mp.yaml

```diff
@@ -0,0 +1,2 @@
+nar-n-iterations: 10
```

Comment:
```suggestion
mp-n-iterations: 10
```

Reply:
@sw005320 I was thinking of using `nar-n-iterations` for other nar models. Should I still make it specific to the mask-predict model?

Reply:
let's make it mask-predict specific for now.
I'm thinking of the more modularized implementation of this one in espnet2 in the near future.